ycsb-mongodb/mongodb/pom.xml

```diff
@@ -24,6 +24,16 @@
       <artifactId>core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Do we want to leave a link to the [driver docs](https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/connection/network-compression/) somewhere ?

Reply:
Could add the comment / link to the XML file too.

Reply:
added comments to both xmls